internal/coreinternal/golden/sort_metrics.go

```diff
@@ -51,6 +52,7 @@ func sortMetrics(ms pmetric.Metrics) {
 			}
 		}
 	}
+	sortMetricDataPointSlices(ms)
```

Comment:
```suggestion
```
Let's also rename `sortMetrics` to `sortMetricAttributes`. This way, we can call one or both of the functions in either order. (This becomes important later if we turn these into options.)